src/components/ToDo.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./ToDo.css";
+const ToDo = ({ todo, handleToggle }) => {
+  const onCheckBoxClick = (todo, e) => {
+    todo.complete = !todo.complete;
+    handleToggle(e.currentTarget.id);
+  };
+  return (
+    <div>
+      <input
+        id={todo.id}
+        key={todo.id + todo.task}
```

Comment:
what is the purpose of id and key?

Reply:
Added id and key considering it could be used in future code changes. Removing them for now.